### PR TITLE
Bugfix: Remove nested form from InputChips

### LINKED ISSUE #2711: Svelte v5 RC: InputChip component produces errors
### Current Behavior

I have a weird error with the InputChip. It doesn't work when it is a child of a `<div>` and a child of a `<form>`.

**Note:** This error occurs only in **Svelte 5 RC**, not in Svelte 4.

This works ✅:

```ts
<script>
	import { InputChip } from '@skeletonlabs/skeleton';
	let list = ['foo', 'bar', 'fizz', 'buzz'];
</script>

<InputChip bind:value={list} name="chips" />
```

This works ✅:

```ts
<script>
	import { InputChip } from '@skeletonlabs/skeleton';
	let list = ['foo', 'bar', 'fizz', 'buzz'];
</script>

<div>
  <InputChip bind:value={list} name="chips" />
</div>
```

This works ✅:

```ts
<script>
	import { InputChip } from '@skeletonlabs/skeleton';
	let list = ['foo', 'bar', 'fizz', 'buzz'];
</script>

<form>
  <InputChip bind:value={list} name="chips" />
</form>
```

This does not work ❌:

```ts
<script>
	import { InputChip } from '@skeletonlabs/skeleton';
	let list = ['foo', 'bar', 'fizz', 'buzz'];
</script>

<form>
  <div>
    <InputChip bind:value={list} name="chips" />
  </div>
</form>
```

I get the following errors:

```
client.js?v=7c866b2e:299 Uncaught (in promise) Svelte error: svelte_component_invalid_this_value
The `this={...}` property of a `<svelte:component>` must be a Svelte component, if defined

	in root.svelte

    at svelte_component_invalid_this_value (http://localhost:5174/node_modules/.vite/deps/chunk-MVFJIXHO.js?v=7c866b2e:385:19)
    at Module.validate_dynamic_component (http://localhost:5174/node_modules/.vite/deps/chunk-ZC2UNLWN.js?v=7c866b2e:2813:7)
    at http://localhost:5174/.svelte-kit/generated/root.svelte:74:7
    at http://localhost:5174/node_modules/.vite/deps/chunk-ZC2UNLWN.js?v=7c866b2e:1035:30
    at execute_reaction_fn (http://localhost:5174/node_modules/.vite/deps/chunk-MVFJIXHO.js?v=7c866b2e:1609:29)
    at execute_effect (http://localhost:5174/node_modules/.vite/deps/chunk-MVFJIXHO.js?v=7c866b2e:1733:20)
    at create_effect (http://localhost:5174/node_modules/.vite/deps/chunk-MVFJIXHO.js?v=7c866b2e:488:7)
    at branch (http://localhost:5174/node_modules/.vite/deps/chunk-MVFJIXHO.js?v=7c866b2e:594:10)
    at http://localhost:5174/node_modules/.vite/deps/chunk-ZC2UNLWN.js?v=7c866b2e:1035:17
    at execute_reaction_fn (http://localhost:5174/node_modules/.vite/deps/chunk-MVFJIXHO.js?v=7c866b2e:1609:29)Svelte error: svelte_component_invalid_this_value
The `this={...}` property of a `<svelte:component>` must be a Svelte component, if defined

	in root.svelte

    at svelte_component_invalid_this_value (http://localhost:5174/node_modules/.vite/deps/chunk-MVFJIXHO.js?v=7c866b2e:385:19)
    at Module.validate_dynamic_component (http://localhost:5174/node_modules/.vite/deps/chunk-ZC2UNLWN.js?v=7c866b2e:2813:7)
    at http://localhost:5174/.svelte-kit/generated/root.svelte:74:7
    at http://localhost:5174/node_modules/.vite/deps/chunk-ZC2UNLWN.js?v=7c866b2e:1035:30
    at execute_reaction_fn (http://localhost:5174/node_modules/.vite/deps/chunk-MVFJIXHO.js?v=7c866b2e:1609:29)
    at execute_effect (http://localhost:5174/node_modules/.vite/deps/chunk-MVFJIXHO.js?v=7c866b2e:1733:20)
    at create_effect (http://localhost:5174/node_modules/.vite/deps/chunk-MVFJIXHO.js?v=7c866b2e:488:7)
    at branch (http://localhost:5174/node_modules/.vite/deps/chunk-MVFJIXHO.js?v=7c866b2e:594:10)
    at http://localhost:5174/node_modules/.vite/deps/chunk-ZC2UNLWN.js?v=7c866b2e:1035:17
    at execute_reaction_fn (http://localhost:5174/node_modules/.vite/deps/chunk-MVFJIXHO.js?v=7c866b2e:1609:29)
chunk-ZC2UNLWN.js?v=7c866b2e:1202 Uncaught TypeError: dom.getAttribute is not a function
    at Array.remove_defaults (chunk-ZC2UNLWN.js?v=7c866b2e:1202:25)
    at run_all (chunk-DKI5ZJQF.js?v=7c866b2e:12:11)
    at process_idle_tasks (chunk-MVFJIXHO.js?v=7c866b2e:1178:3)
```

### Expected Behavior

_No response_

### Steps To Reproduce

_No response_

### Link to Reproduction / Stackblitz

https://stackblitz.com/edit/vitejs-vite-ijoavm?file=src%2Froutes%2F%2Blayout.svelte&terminal=dev

### More Information

_No response_

### PR DESCRIPTION
**Pull request by endigo9740**
## Linked Issue

Closes #2711

## Description

This replaces the nested form used to capture mobile `Enter` keys with a `on:keyup` event handler.

NOTE: this will need to be tested heavily in the world world before we can consider merging, as it has the potential to be a breaking change.

## Changsets

Instructions: Changesets automate our changelog. If you modify files in `/packages`, run `pnpm changeset` in the root of the monorepo, follow the prompts, then commit the markdown file. Changes that add features should be `minor` while chores and bugfixes should be `patch`. Please prefix the changeset message with `feat:`, `bugfix:` or `chore:`.

## Checklist

Please read and apply all [contribution requirements](https://www.skeleton.dev/docs/contributing).

- [x] This PR targets the `dev` branch (NEVER `master`)
- [x] Documentation reflects all relevant changes
- [x] Branch is prefixed with: `docs/`, `feat/`, `chore/`, `bugfix/`
- [ ] Ensure Svelte and Typescript linting is current - run `pnpm ci:check`
- [ ] Ensure Prettier linting is current - run `pnpm format`
- [ ] All test cases are passing - run `pnpm test`
- [x] Includes a changeset (if relevant; see above)

### CODE REVIEW

File: packages/skeleton/src/lib/components/InputChip/InputChip.svelte [hunk 1]
```diff
@@ -223,21 +219,24 @@
...
-	function addChipInternally(event: SvelteEvent<SubmitEvent, HTMLFormElement>): void {
-		event.preventDefault();
-		// Validate
-		inputValid = validate();
-		// When the onInvalid hook is present
-		if (inputValid === false) {
-			/** @event {{ event: Event, input: any  }} invalid - Fires when the input value is invalid. */
-			dispatch('invalid', { event, input });
-			return;
+	function onKeyUpHandler(event: KeyboardEvent): void {
```
Comment (Hugos68) [inline packages/skeleton/src/lib/components/InputChip/InputChip.svelte:222]: Change to `onKeyDownHandler`

File: packages/skeleton/src/lib/components/InputChip/InputChip.svelte [hunk 2]
```diff
@@ -223,21 +219,24 @@
...
+		if (event.key === 'Enter') {
+			event.preventDefault();
+			event.stopPropagation();
```
Comment (Hugos68) [inline packages/skeleton/src/lib/components/InputChip/InputChip.svelte:225]: You can remove this line

File: packages/skeleton/src/lib/components/InputChip/InputChip.svelte [hunk 3]
```diff
@@ -294,19 +293,17 @@
...
-				on:blur
-				disabled={$$restProps.disabled}
-			/>
-		</form>
+		<input
+			type="text"
+			bind:value={input}
+			placeholder={$$restProps.placeholder ?? 'Enter values...'}
+			class="input-chip-field {classesInput}"
+			on:keyup={onKeyUpHandler}
```
Comment (Hugos68) [inline packages/skeleton/src/lib/components/InputChip/InputChip.svelte:301]: Change to `on:keydown`

File: packages/skeleton/src/lib/components/InputChip/InputChip.svelte [hunk 4]
```diff
@@ -223,21 +219,24 @@
...
-		event.preventDefault();
-		// Validate
-		inputValid = validate();
-		// When the onInvalid hook is present
-		if (inputValid === false) {
-			/** @event {{ event: Event, input: any  }} invalid - Fires when the input value is invalid. */
-			dispatch('invalid', { event, input });
-			return;
+	function onKeyUpHandler(event: KeyboardEvent): void {
+		if (event.key === 'Enter') {
```
Comment (Hugos68) [inline packages/skeleton/src/lib/components/InputChip/InputChip.svelte:223]: Bit of a nitpick, but I would just use and early return: `if (event.key !== 'Enter') return`

**Review by Hugos68 — CHANGES_REQUESTED** [review]

**Review by Hugos68 — APPROVED** [review]

**Comment by multiwebinc** [comment]
Are the changes to all the `pnpm-lock` files necessary for this PR?

**Comment by niktek** [comment]
> Are the changes to all the `pnpm-lock` files necessary for this PR?

Could they have been in a separate PR, yes, but they aren't unexpected either as they are a result of updating to pnpm v9

**Comment by endigo9740** [comment]
Yeah sorry about that. I work on multiple projects and recently needed to update PNPM for one of those. This was just a side effect.

**Pull request merged**